test.properties

```diff
@@ -8414,4 +8414,4 @@
     vm.cluster-node-2.ip=
 
     vm.db.host=vm-3
-    vm.db.ip=
\ No newline at end of file
+    vm.db.ip=
```

Comment:
Hey @calvinctvo, could you discard this change?

Reply:
done! I just revert back to this commit 432fba8e2dda42a032aa8abd9144d1fa7662a581